src/elements/footer-nav.html

```diff
@@ -0,0 +1,94 @@
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../../bower_components/iron-image/iron-image.html">
+
+
+<dom-module id="footer-nav">
+  <template>
+    <style is="custom-style" include="shared-styles flex flex-alignment"></style>
+
+    <style>
+
+      .nav-inline {
+        list-style: none;
+        margin: 0;
+        padding: 0;
+      }
+
+      .footer-logo {
+        margin: 10px 30px 0 0;
+        height: 24px;
+        width: 120px;
+        float: left;
+      }
+
+      .copyright {
+        margin-right: 0;
+        padding: 15px 0 0 0;
+        box-sizing: border-box;
+        float: right;
+      }
+
+      .cod {
+        margin-right: 0;
+        padding: 15px 0 0 0;
+        float: left;
+      }
+
+      a {
+        color: var(--footer-text-color);
+        padding-bottom: 2px;
+        text-decoration: none;
+      }
+
+      a:hover {
+        text-decoration: underline;
+      }
+
+      @media (max-width: 811px) {
```

Comment:
We have mobile first development. So `@media` should be used with `(min-width: ...)`